### PR TITLE
Module 1:Day 7 - Portfolio Project

### PR DESCRIPTION
**Pull request by Strangeal**
### Contact form

Added contact form in HTML file 
Added contact form styling in CSS file

### CODE REVIEW

File: header.html [hunk 1]
```diff
@@ -146,6 +146,18 @@ <h2>About Myself</h2>
...
+            <section class="form-group">
+                <img src="img/contact-form-shapes-mobile.svg" alt="Background shapes design" class="form-bg-img">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it.</p>
+                <form class="form-input">
+                    <input type="text" name="text" id="name" placeholder="Enter your name" maxlength="30" required>
+                    <input type="email" name="Email" id="email" placeholder="Enter your email" required>
+                    <textarea name="comments" id="comments" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
+                    <button class="btn" type="submit">Get It Touch</button>
+                </form>
```
Comment (prolajumokeoni) [inline header.html]: - [x] Kindly add the [form service](https://formspree.io/html/) to collect data from forms, please check the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) to see how to integrate this feature,

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Strangeal 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: header.html [hunk 2]
```diff
@@ -146,6 +146,18 @@ <h2>About Myself</h2>
...
+            <!-- Contact form -->
+            <section class="form-group">
+                <img src="img/contact-form-shapes-mobile.svg" alt="Background shapes design" class="form-bg-img">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it.</p>
+                <form class="form-input">
+                    <input type="text" name="text" id="name" placeholder="Enter your name" maxlength="30" required>
+                    <input type="email" name="Email" id="email" placeholder="Enter your email" required>
+                    <textarea name="comments" id="comments" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
+                    <button class="btn" type="submit">Get It Touch</button>
```
Comment (prolajumokeoni) [inline header.html:158]: - [x] Kindly remove the border button outline from the Get in touch button

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @Strangeal 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

- [ ] [OPTIONAL] It would be better to add padding to all inputs in order to match the Figma design, also you should change the color of the inputs text and  font-size, I encourage you to do that
- FIGMA 
Screen Shot 2022-08-23 at 10 58 14
- Yours
Screen Shot 2022-08-23 at 11 17 23


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**